plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/JdbcColumnHandleUtil.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.jdbc;
+
+import io.trino.spi.type.Type;
+
+public final class JdbcColumnHandleUtil {
+    private JdbcColumnHandleUtil() {
+    }
+
+    public static JdbcColumnHandle getJdbcColumnHandle(String columnName, JdbcTypeHandle jdbcTypeHandle, Type columnType) {
```

Comment:
move this method to `JdbcColumnHandle` and rename it to: `createJdbcColumnHandle`

Reply:
done